timezone_converter/comparison_view.py

```diff
@@ -53,11 +53,21 @@ def _build_table(self) -> Table:
             table.add_column(header, justify='center')
 
         fmt = '%Y-%m-%d %H:%M'
+
+        current_hour = datetime.now().hour
         for hour in range(24):
-            columns = [
-                (midnight + timedelta(hours=hour)).strftime(fmt)
-                for midnight in self.midnights
-            ]
+            if hour != current_hour:
```

Comment:
I have some suggestions:

- I prefer that we use a default value and then check if the hour matches to then change the value
- We can use `f-strings`
- My color suggestion would be blue, as red is usually reserved for errors, and the opposite for green (which we may end up using later for other features)

This is the how the code would end up looking:

```py
columns = [
    (midnight + timedelta(hours=hour)).strftime(fmt)
    for midnight in self.midnights
]
if hour == current_hour:
    columns = [
        f'[blue]{(midnight + timedelta(hours=hour)).strftime(fmt)}[/blue]'
        for midnight in self.midnights
    ]

```